src/components/atoms/OakIcon/OakIcon.tsx

```diff
@@ -25,6 +25,15 @@ export function isValidIconName(iconName: string): iconName is OakIconName {
   return oakIconNames.includes(iconName as OakIconName);
 }
 
+/**
+ * returns a Icon URL from Cloudinary if is a valid icon, otherwise returns undefined
+ */
+export function generateOakIconURL(iconName: string) {
```

Comment:
We should also use this function in the `src` property below (ln54)

Reply:
Yep, makes sense. Is there an Icon / string we can default to if it isn't a valid Icon name?

Reply:
The `books` icon was agreed upon as a fallback for subject icons, would that work here?